app/templates/custom-elements/remote-screen.html

```diff
@@ -199,6 +190,107 @@
           screenElement.addEventListener("mousedown", (evt) => {
             this.rateLimitedMouse.onMouseDown(evt);
           });
+          //-------------------------------------------------------------------------------
+          var isPointerMoving = false;
```

Comment:
We're adding a lot of touch-specific logic to the remote screen component, which I'd like to avoid because there's already a lot of complexity here.

Could we refactor this into a separate file like `touchHandler.js` (similar to `app/static/js/mouse.js`). And then, we could reduce the logic in `remote-screeen.html` to something like:

```javascript
screenElement.addEventListener("touchstart", (evt) => {
  this.touchHandler.onTouchStart(evt);
});
screenElement.addEventListener("touchend", (evt) => {
  this.touchHandler.onTouchEndevt);
});
...
```

That way, we could encapsulate all the variables that we only need for touch logic to just live in `touchHandler.js`.

Reply:
refactored.